ceph_deploy/hosts/suse/mon/create.py

```diff
@@ -6,14 +7,50 @@ def create(distro, args, monitor_keyring):
     hostname = distro.conn.remote_module.shortname()
     common.mon_create(distro, args, monitor_keyring, hostname)
 
-    remoto.process.run(
-        distro.conn,
-        [
-            'rcceph',
-            '-c',
-            '/etc/ceph/{cluster}.conf'.format(cluster=args.cluster),
-            'start',
-            'mon.{hostname}'.format(hostname=hostname)
-        ],
-        timeout=7,
-    )
+    if distro.init == 'sysvinit':
+        service = distro.conn.remote_module.which_service()
+        remoto.process.run(
+            distro.conn,
+            [
+                service,
+                'ceph',
+                '-c',
+                '/etc/ceph/{cluster}.conf'.format(cluster=args.cluster),
+                'start',
+                'mon.{hostname}'.format(hostname=hostname)
+            ],
+            timeout=7,
+        )
+
+        system.enable_service(distro.conn)
+    elif distro.init == 'systemd':
+       # enable ceph target for this host (in case it isn't already enabled)
```

Comment:
I could not find an explicit documentation stating that you can enable the same service N times. But I did find that it will create symlinks to persist the information of enabling a service, and will not start a service when enabling.

We have an exception to this when the parameters of the parametrised service is different but this is as would be expected.